src/components/Alertbox/Alertbox.scss

```diff
@@ -4,6 +4,7 @@
   background-color: $error;
   color: $textonprimary;
   font-size: 19px;
+  line-height: 1.25em;
```

Comment:
To reduce line height on the text content shouldn't this rule be applied to

```css
.dfo-alertbox__content p {
  line-height: 1.25em;
}
```

?

Reply:
Appears right!